### PR TITLE
Updated lists of SUSE Linux Enterprise hardening profiles in openSCAP

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/24627 <== **there in the description you can find a link to the rendered output**

https://jira.suse.com/browse/DOCTEAM-1428

- master
- 5.0
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Updated lists of SUSE Linux Enterprise hardening profiles in openSCAP
+  chapter of the Administration Guide
```
Comment (0rnela) [inline CHANGELOG.md:2]:
```suggestion
- Updated lists of SUSE Linux Enterprise hardening profiles in openSCAP
  chapter in the Administration Guide
```

**Review by 0rnela — APPROVED** [review]

File: modules/administration/pages/openscap.adoc [hunk 2]
```diff
@@ -71,13 +71,38 @@ You are advised to use the [package]``scap-security-guide-redhat`` package you w
...
 ====
 {suse} provides the [package]``scap-security-guide`` package for different openscap profiles.
-In the current version of [package]``scap-security-guide``, {suse} supports the following profiles:
-
-* DISA STIG profile for {sles} 12 and 15
-* PCI-DSS profile for {sles} 12 and 15
-* HIPAA profile for {sles} 12 and 15
-
-Other profiles, like the CIS profile, are community supplied and not officially supported by {suse}.
+In the current version of [package]``scap-security-guide``, {suse} supports the following profiles for {sles} 15:
```
Comment (admd) [inline modules/administration/pages/openscap.adoc:74]: Keep the existing format and use the following list

- DISA STIG profile for SUSE Linux Enterprise Server 12 and 15
- ANSSI-BP-028 profile for SUSE Linux Enterprise Server 12 and 15
- PCI-DSS profile for SUSE Linux Enterprise Server 12 and 15
- HIPAA profile for SUSE Linux Enterprise Server 12 and 15
- CIS profile for SUSE Linux Enterprise Server 12 and 15
- Hardening for Public Cloud Image of SUSE Linux Enterprise Server for SAP Applications 15
- Public Cloud Hardening for SUSE Linux Enterprise 15

File: modules/administration/pages/openscap.adoc [hunk 3]
```diff
@@ -71,13 +71,38 @@ You are advised to use the [package]``scap-security-guide-redhat`` package you w
...
+* ANSSI-BP-028 (minimal)
+* CIS SUSE SUSE Linux Enterprise 12 Benchmark Level 2 (Workstation)
+* CIS SUSE Linux Enterprise 12 Benchmark for Level 1 (Server)
+* CIS SUSE Linux Enterprise 12 Benchmark for Level 1 (Workstation)
+* CIS SUSE Linux Enterprise 12 Benchmark for Level 2 (Server)
+* DISA STIG for SUSE Linux Enterprise 12
+* PCI-DSS v.4 Control Baseline for SUSE Linux Enterprise 12
+* Standard System Security Profile for SUSE Linux Enterprise 12
+
+// Other profiles, like the CIS profile, are community supplied and not officially supported by {suse}.
```
Comment (admd) [inline modules/administration/pages/openscap.adoc:105]:
```suggestion
// Any other profiles are community supplied and not officially supported by {suse}.
```

File: modules/administration/pages/openscap.adoc [hunk 4]
```diff
@@ -74,10 +74,14 @@ You are advised to use the [package]``scap-security-guide-redhat`` package you w
 In the current version of [package]``scap-security-guide``, {suse} supports the following profiles:
 
 * DISA STIG profile for {sles} 12 and 15
+* ANSSI-BP-028 profile for {sles} 12 and 15
 * PCI-DSS profile for {sles} 12 and 15
 * HIPAA profile for {sles} 12 and 15
```
Comment (sounix000) [inline modules/administration/pages/openscap.adoc:79]: HIPPA is available only for SLES 15, and not SLES12.

File: modules/administration/pages/openscap.adoc [hunk 5]
```diff
@@ -74,10 +74,14 @@ You are advised to use the [package]``scap-security-guide-redhat`` package you w
...
+* CIS profile for {sles} 12 and 15
+* Hardening for Public Cloud Image of {sles} for SAP Applications 15
+* Public Cloud Hardening for {sle} 15
```
Comment (sounix000) [inline modules/administration/pages/openscap.adoc:83]: Standard System Security profile for SLES 15 and 12 is missing from the list.

References (see top right drop-down in the following pages):
- https://static.open-scap.org/ssg-guides/ssg-sle15-guide-index.html
- https://static.open-scap.org/ssg-guides/ssg-sle12-guide-index.html

**Pull request merged**